Dockerfile

```diff
@@ -14,6 +14,7 @@ LABEL MAINTAINERS="Nik Everett <nik@elastic.co>"
 #   * openssh-client (used by git)
 #   * openssh-server (used to forward ssh auth for git when running with --all on macOS)
 #   * perl-base
+#   * python (is python2)
```

Comment:
This reminds me. Is there an issue for changing to python 3, and explicitly weeding out and dropping support for python 2 wherever it may be found? Because python 2 has less than a year to live. I didn't see one on a quick issue search, but I might have just missed it.

Reply:
Not *really*, no. But, hey, I'll make [one](https://github.com/elastic/docs/issues/636).